Failed to build docs.rs page for ratatui-widgets

API documentation for `ratatui-widgets` (beta) is not available: https://docs.rs/crate/ratatui-widgets/0.3.0-beta.0

```
[INFO] running `Command { std: "docker" "start" "-a" "a655db6a36aaf614ee66afa9aa4719c378e18405572fd22f9039f0d30842d7ff", kill_on_drop: false }`
[INFO] [stderr]     Scraping ratatui-widgets v0.3.0-beta.0 (/opt/rustwide/workdir)
[INFO] [stderr] error[E0433]: failed to resolve: use of unresolved module or unlinked crate `ratatui`
[INFO] [stderr]   --> examples/barchart.rs:20:5
[INFO] [stderr]    |
[INFO] [stderr] 20 | use ratatui::layout::{Constraint, Layout, Rect};
[INFO] [stderr]    |     ^^^^^^^ use of unresolved module or unlinked crate `ratatui`
[INFO] [stderr]    |
[INFO] [stderr]    = help: if you wanted to use a crate named `ratatui`, use `cargo add ratatui` to add it to your `Cargo.toml`
```

See the full logs: https://docs.rs/crate/ratatui-widgets/0.3.0-beta.0/builds/2624165

I believe this is due to [scrape-examples](https://doc.rust-lang.org/rustdoc/scraped-examples.html) feature because it tries to run the examples inside `ratatui-widgets` which depend on `ratatui` (which is not found / circular dependency?).

This works locally via `cargo +nightly doc -Zunstable-options -Zrustdoc-scrape-examples` because `ratatui` is a `dev-dependency` and it works _somehow_.

--- a/ratatui-widgets/Cargo.toml
+++ b/ratatui-widgets/Cargo.toml
@@ -68,7 +68,7 @@ unicode-width.workspace = true
 color-eyre.workspace = true
 crossterm.workspace = true
 pretty_assertions.workspace = true
-ratatui = { path = "../ratatui" }
+ratatui.workspace = true
 rstest.workspace = true
 
 [lints]

